File: front-end/src/app/model.ts/flight.ts
import { Origin } from './origin';
import { Destination } from './destination';
import { Duration } from './duration';

export class Flight {
    flightNum: string;
    origin: Origin;
    destination: Destination;
<<<<<<< HEAD
    duration: Duration;
=======
    duration: Duration; 
>>>>>>> bb0fbabea9d8a5b0fb82eb8487f4ddf7d362e50e
    aircraftModel: string;
    
    constructor(flight: Flight){
        this.flightNum = flight.flightNum;
        this.origin = flight.origin;
        this.destination = flight.destination;
        this.duration = flight.duration;
        this.aircraftModel = flight.aircraftModel;
    }
}

File: back-end/src/main/java/com/aa/improvinghackathon/infrastructure/service/flights/response/Flight.java
package com.aa.improvinghackathon.infrastructure.service.flights.response;

public class Flight {

    private String flightNumber;
    private Location origin;
    private Location destination;
    private Duration duration;
    private Aircraft aircraft;

    public String getFlightNumber() {
        return flightNumber;
    }

    public void setFlightNumber(String flightNumber) {
        this.flightNumber = flightNumber;
    }

    public Location getOrigin() {
        return origin;
    }

    public void setOrigin(Location origin) {
        this.origin = origin;
    }

    public Location getDestination() {
        return destination;
    }

    public void setDestination(Location destination) {
        this.destination = destination;
    }

    public Duration getDuration() {
        return duration;
    }

    public void setDuration(Duration duration) {
        this.duration = duration;
    }

    public Aircraft getAircraft() {
        return aircraft;
    }

    public void setAircraft(Aircraft aircraft) {
        this.aircraft = aircraft;
    }

}
